config/routes.rb

```diff
@@ -29,6 +30,14 @@
       post :select
     end
 
+    #JE Todo: Confirm if we should nest these internal API routes under a namespace like "API::V1".
+    #JE Todo: Confirm if there is a better way to define these routes.
+    namespace :calendar_events do
+      root to: :index
+      get 'loan/:loan_id', to: :loan
+      get 'division/:division_id', to: :division
+    end
```

Comment:
I'd actually started with the query param permutations, but thought that RESTful style URLs were more appropriate for an API.  I've saved out that code into a .scratch file (my private stash for dead code), so if we do later want to support a stateless event feed for a division scope it'll be handy.